themes/digital.gov/src/scss/new/_topics-button.scss

```diff
@@ -0,0 +1,88 @@
+@use "uswds-core" as *;
+
+.topics-button {
+  margin-bottom: units("105");
+
+  &__link {
+    @include u-font("sans", "sm");
+    @include u-text("medium");
+    @include u-text("no-underline");
+    @include u-bg("white");
+    @include u-text("primary-vivid");
+    @include u-border("1px", "primary-vivid", "solid");
+    @include u-radius("sm");
+    display: block;
+    overflow: auto;
+    padding: units(1);
+    transition: background-color 0.5s, color 0.5s;
+    white-space: nowrap;
+
+    @include at-media("tablet") {
+      padding-left: units(1);
+    }
+  }
+
+  &__arrow {
+    @include u-float("right");
+    @include u-text("gray-1");
+    display: none;
+    margin-right: 10px;
+    transition: margin 0.4s;
+
+    @include at-media("tablet-lg") {
+      display: block;
+    }
+  }
+
+  &__list {
+    list-style: none;
+    padding-left: 0;
+
+    @include at-media("tablet-lg") {
+      column-count: 3;
+    }
+  }
+
+  &__number {
+    @include u-padding-x("05");
+    @include u-radius("pill");
+    @include u-bg("primary-lightest");
+    @include u-text("base-darker");
+    @include u-font("sans", "2xs");
+    height: fit-content;
+    margin: auto;
+  }
+}
+
+.topics-button--primary {
+  .topics-button__link {
+    &:hover {
+      @include u-bg("primary-vivid");
+      @include u-text("gray-1");
+      @include u-border("1px", "primary-vivid", "solid");
+      span {
+        margin-right: 0;
+      }
+    }
+  }
+}
+
+.topics-button--count {
+  display: flex;
+  list-style: none;
+  margin-bottom: 0;
+  transition: none;
+  width: fit-content;
+
+  .topics-button__link {
+    border: none;
+
+    &:visited {
+      color: #5942d2;
```

Comment:
Can we use a USWDS color token here?

Reply:
yes... nick just showed me a cool tool that can help id a uswds color token based on the color mark up, updated with closest one